package.json

```diff
@@ -1,8 +1,9 @@
 {
+  "private": true,
   "scripts": {
     "dev": "docker compose -f docker-compose.dev.yml up -d --force-recreate --build --wait",
     "dev:attach": "docker compose -f docker-compose.dev.yml up --force-recreate --build",
-    "dev:down": "docker compose -f docker-compose.dev.yml down --remove-orphans",
+    "dev:down": "docker compose -f docker-compose.dev.yml down -v --rmi all --remove-orphans",
```

Comment:
This command was leaving behind named volumes and images.

Reply:
Maybe this resolves #340 as well!

Reply:
Yes it does. I mean, if the command is used every time. In case someone drops container stacks manually but leaves images or volumes - they will stay there until deleted manually as well. Otherwise, the command will clean everything that docker compose creates. :)